homeassistant/components/dnsip/const.py

```diff
@@ -16,4 +17,6 @@
 DEFAULT_IPV6 = False
 DEFAULT_NAME = "myip"
 DEFAULT_RESOLVER = "208.67.222.222"
+DEFAULT_PORT = 53
```

Comment:
We could just use the same constant right?

Reply:
We could. I didn't because I couldn't find an RFC that said 53 is definitely the default port for IPV6 and its not a fully implemented standard so I wanted to give scope for things to be easily changed.
That being said, it wouldn't be hard to make the change in the very unlikely event it changes to something other that 53 so I will make your suggested change

Reply:
Done in [fabfea5](https://github.com/home-assistant/core/pull/113993/commits/fabfea5c057d77df97f31a8b691f75cec58a7227)